invisible_cities/cities/penthesilea_test.py

```diff
@@ -9,7 +9,7 @@
 from .. io                   import dst_io as dio
 
 
-def test_penthesilea_run_on_Kr_MC_ipmt_pmaps_5evt(ICDIR, config_tmpdir):
+def test_penthesilea_run_on_Kr_MC_ipmt_pmaps_6evt(ICDIR, config_tmpdir):
```

Comment:
This is already being changed in another PR, where the "ipmt" name is removed, as the test has nothing to do with "ipmts"

Reply:
The "ipmt" is included because the name of the test is identifying the name of the input file in `test_data` that the test is testing.